includes/payment-methods/class-wechatpay-payment-method.php

```diff
@@ -55,4 +93,62 @@ public function get_title( ?string $account_country = null, $payment_details = f
 	public function get_testing_instructions( string $account_country ) {
 		return '';
 	}
+
+	/**
+	 * Returns payment method supported currencies for the merchant's account.
+	 *
+	 * @return array
+	 */
+	public function get_currencies() {
```

Comment:
I'm not sure if this implementation is correct. What I understood from the [Stripe article](https://docs.stripe.com/payments/wechat-pay/accept-a-payment?ui=direct-api#supported-currencies) is that CNY is supported in all countries, plus that country domestic currency. So, for example, in the US the supported currencies would be both CNY and USD.

Reply:
Nice catch! I also noticed that a few European countries support their local currency as well. Fixed with [f4dece0](https://github.com/Automattic/woocommerce-payments/pull/10381/commits/f4dece05ecf343f83c582ff2a517ca945041f5fd)